utilities/app/src/test/java/utilities/MultiBracketValidationTester.java

```diff
@@ -0,0 +1,26 @@
+package utilities;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class MultiBracketValidationTester {
```

Comment:
Good work with your tests! It's good to have multiple tests for all the cases you might run into. Does it return false if the input given is an empty string?